### ISSUE TITLE
Provide some documentation on how to find and install SqlWatchImport.exe

### ISSUE THREAD

**Issue by sqllensman** [marcingminski/sqlwatch#393]
The current documentation for using SqlWatchImport.exe is extremely sparse and missing the most basic detail like "What files need to be installed and where are they found?

Does this file get installed when running the Install-DbaSqlWatch ?
If so where?

If I look at the locally downloaded files at C:\Users\XXXX\AppData\Roaming\PowerShell\dbatools\4.2\SQLWATCH 4.2.0.37410 20210518204739 I can see a SqlWatchImport.exe but there is no indication of what other files are required.

**Comment by marcingminski** [comment]
Just run it. You do not need to install anything. 
Files are found in the release, like all the other files.

**Comment by sqllensman** [comment]
SQLWatch has been installed onto a monitoring Server using an Internet Connected Workstation

The release files appear only to be downloaded to  C:\Users\XXXX\AppData\Roaming\PowerShell\dbatools\4.2\SQLWATCH 4.2.0.37410 (a hidden directory that will change with every release) on the workstation.

The SqlWatchImport.exe needs to be run on the remote Server via SQL Agent using a suitable proxy account.

It is not possible or practical to "Just run it". as the files are on different Server.
I was able to get it running by copying most of the files into a new directory on the remote Server however it unclear if all these files are required.

**Comment by marcingminski** [comment]
Fair enough, good shout. Would you like to help update the documentation with your findings? It's all on GitHub with the aim that everyone can contribute. Thanks!

**Comment by sqllensman** [comment]
No problem.

I will create a Pull request over the next few days

**Comment by marcingminski** [comment]
Thank you kindly.
